doc/user/layouts/shortcodes/postgres-direct/right-size-the-cluster.html

```diff
@@ -26,10 +26,22 @@
     (1 row)
     ```
 
-1. Going forward, you can verify that your new replica size is sufficient by checking the replica lag in your PostgreSQL database:
+1. Going forward, you can verify that your new replica size is sufficient as follows:
 
-    ```sql
-    SELECT extract(epoch from now() - pg_last_xact_replay_timestamp()) AS replica_lag;
-    ```
+    1. In Materialize, get the replication slot name associated with your PostgreSQL source from the [`mz_internal.mz_postgres_sources`](/sql/system-catalog/mz_internal/#mz_postgres_sources) table:
+
+        ```sql
+        SELECT * FROM mz_internal.mz_postgres_sources;
+        ```
+
+    1. In PostgreSQL, check the replica lag, using the replication slot name from the previous step:
```

Comment:
Rather than "replica lag", let's say "replication slot lag." "Replica" in Materialiaze is a term that we use to talk about the physical components that run in a cluster.

I might also, after the query, provide some context about how to interpret this number, e.g.

> The result of this query is the amount of data your Postgres cluster must retain in its WAL because of this replication slot. Typically, this means Materialize has not yet communicated back to Postgres that it has committed this data.

Or something to that effect.